OpenRA.Mods.Common/Traits/HitShape.cs

```diff
@@ -19,6 +19,12 @@ namespace OpenRA.Mods.Common.Traits
 	[Desc("Shape of actor for damage calculations.")]
 	public class HitShapeInfo : ConditionalTraitInfo
```

Comment:
Now that I think about it, is it actually correct to require `BodyOrientation` here? I mean, bridges etc. don't need a `BodyOrientation` otherwise, I could instead adapt the `TargetablePositions` code below to only rotate the offset if the actor has a `BodyOrientation`.